pkg/loop/internal/core/services/capability/standard_capabilities.go

```diff
@@ -33,7 +34,7 @@ type StandardCapabilities interface {
 type StandardCapabilitiesClient struct {
 	*goplugin.PluginClient
 	capabilitiespb.StandardCapabilitiesClient
-	*baseCapabilityClient
+	// *baseCapabilityClient
```

Comment:
Not sure if this is used or not, would need to make the types public otherwise

Reply:
I think this is probably safe to remove, @ettec would you agree?